OmiseSDK/AtomeFormViewController.swift

```diff
@@ -0,0 +1,322 @@
+    @IBOutlet private var shippingCityTextField: OmiseTextField!
+    @IBOutlet private var shippingCountryCodeTextField: OmiseTextField!
+    @IBOutlet private var shippingPostalCodeTextField: OmiseTextField!
+    @IBOutlet private var submitButton: MainActionButton!
+    @IBOutlet private var requestingIndicatorView: UIActivityIndicatorView!
+    
+    @IBOutlet private var phoneNumberErrorLabel: UILabel!
+    @IBOutlet private var emailErrorLabel: UILabel!
+    @IBOutlet private var shippingStreetErrorLabel: UILabel!
+    @IBOutlet private var shippingCityErrorLabel: UILabel!
+    @IBOutlet private var shippingCountryErrorLabel: UILabel!
+    @IBOutlet private var shippingPostalCodeErrorLabel: UILabel!
+    
+    @IBOutlet var formLabels: [UILabel]!
+    @IBOutlet var formFields: [OmiseTextField]!
+    
+    @IBOutlet var formFieldsAccessoryView: UIToolbar!
+    @IBOutlet var gotoPreviousFieldBarButtonItem: UIBarButtonItem!
+    @IBOutlet var gotoNextFieldBarButtonItem: UIBarButtonItem!
+    @IBOutlet var doneEditingBarButtonItem: UIBarButtonItem!
+    
+    // need to refactor loadView, removing super results in crash
+    // swiftlint:disable prohibited_super_call
+    override func loadView() {
+        super.loadView()
+        
+        view.backgroundColor = .background
+        formFieldsAccessoryView.barTintColor = .formAccessoryBarTintColor
+        
+        submitButton.defaultBackgroundColor = .omise
+        submitButton.disabledBackgroundColor = .line
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        applyPrimaryColor()
+        applySecondaryColor()
+        navigationItem.backBarButtonItem = UIBarButtonItem(title: "", style: .plain, target: nil, action: nil)
+        
+        formFields.forEach {
+            $0.inputAccessoryView = formFieldsAccessoryView
+        }
+        
+        formFields.forEach {
+            $0.adjustsFontForContentSizeCategory = true
+        }
+        formLabels.forEach {
+            $0.adjustsFontForContentSizeCategory = true
+        }
+        submitButton.titleLabel?.adjustsFontForContentSizeCategory = true
+        
+        if  #available(iOS 11, *) {
+            // We'll leave the adjusting scroll view insets job for iOS 11 and later to the layoutMargins + safeAreaInsets here
+        } else {
+            automaticallyAdjustsScrollViewInsets = true
+        }
+
+        NotificationCenter.default.addObserver(
+            self,
+            selector: #selector(keyboardWillChangeFrame(_:)),
+            name: NotificationKeyboardWillChangeFrameNotification,
+            object: nil
+        )
+        NotificationCenter.default.addObserver(
+            self,
+            selector: #selector(keyboardWillHide(_:)),
+            name: NotificationKeyboardWillHideFrameNotification,
+            object: nil
+        )
+
+        phoneNumberTextField.validator = try? NSRegularExpression(pattern: "\\d{10,11}?", options: [])
```

Comment:
I'm wondering if it'd be confusing if we don't allow the "+" sign but we expect the country code in the field 🤔

Reply:
I agree with you ka. :joy: 
I will add allow "+", and always add "+" if user fill in value without this one before send to api. 🙏

Reply:
Please review again ka. I've changed a little bit from previous reply. This form only allow phone number which start with '+', and add phone number hint for guide user should enter '+' including country code.